src/AgOledDisplay.cpp

```diff
@@ -304,6 +304,10 @@ void OledDisplay::showDashboard(const char *status) {
       DISP()->drawStr(55, 27, "PM2.5");
 
       /** Draw PM2.5 value */
+      int pm25 = value.pm25_1;
+      if (config.hasSensorSHT) {
+        pm25 = ag->pms5003.pm25Compensated(pm25, value.Humidity);
```

Comment:
propose to rename the method to simply .compensate(). We can see from the params that this is related to pm25.

Reply:
Ok, I'll change the name and update API comment to mention the input is PM2.5